docs/lib/INTRO.md

```diff
@@ -0,0 +1,43 @@
+#### uVisor
+
+The uVisor is a self-contained software hypervisor that creates independent secure domains on ARM Cortex&reg;-M3 and Cortex&reg;-M4 microcontrollers. It increases resilience against malware and protects secrets from leaking even among different modules of the same application. You can find a [high-level overview here](http://www.slideshare.net/FoolsDelight/practical-realtime-operating-system-security-for-the-masses) ([Download PDF](https://github.com/ARMmbed/uvisor/raw/docs/uVisorSecurity-TechCon2016.pdf)).
+
+*Note about interrupts:*
+When the uVisor is enabled, all NVIC APIs are rerouted to the corresponding uVisor vIRQ APIs, which virtualize interrupts. The uVisor interrupt model has the following features:
+
+* The uVisor owns the interrupt vector table.
```

Comment:
Does this render correctly on the website? It was my understanding that we had to use `-` instead of `*` for the website to work properly, but perhaps the website tools are more flexible these days.

Reply:
@AnotherButler Can you confirm? I didn't know about this limitation

Reply:
If you wouldn't mind, just for consistency's sake with our other documentation, could you please use `-`?